tests/Facebook/InstantArticles/Transformer/WPTransformerTest.php

```diff
@@ -6,7 +6,7 @@
  * This source code is licensed under the license found in the
  * LICENSE file in the root directory of this source tree.
  */
-namespace Facebook\InstantArticles\Transformer\CMS;
```

Comment:
Folder structure should match the namespace for convenience. I don't see the need to change it here, btw.